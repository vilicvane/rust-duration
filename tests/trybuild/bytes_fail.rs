use lits::bytes;

const _SIZE: u64 = bytes!("not-a-size");

fn main() {}
