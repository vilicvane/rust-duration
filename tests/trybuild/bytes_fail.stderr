error: failed to parse size string: couldn't parse "not-a-size" into a ByteSize, cannot parse float from empty string
 --> tests/trybuild/bytes_fail.rs:3:27
  |
3 | const _SIZE: u64 = bytes!("not-a-size");
  |                           ^^^^^^^^^^^^
